cypress/integration/sucuri-scanner.js

```diff
@@ -123,21 +123,11 @@ describe( 'Run integration tests', () => {
 		cy.get('[data-cy=sucuriscan_row_wp_update_plugins]').find('td:nth-child(3)').contains('quarterly');
 	});
 
-	it('can activate and deactivate the WordPress integrity diff utility', () => {
-		cy.visit('wp-admin/admin.php?page=sucuriscan_settings#scanner');
-
-		cy.get('[data-cy=sucuriscan_scanner_integrity_diff_utility_toggle]').click();
-		cy.get('.sucuriscan-alert').contains('The status of the integrity diff utility has been changed');
-		cy.get('[data-cy=sucuriscan_scanner_integrity_diff_utility_toggle]').contains('Disable');
-
-		cy.get('[data-cy=sucuriscan_scanner_integrity_diff_utility_toggle]').click();
-		cy.get('.sucuriscan-alert').contains('The status of the integrity diff utility has been changed');
-		cy.get('[data-cy=sucuriscan_scanner_integrity_diff_utility_toggle]').contains('Enable');
-	});
-
 	it('can ignore and unignore false positives (integrity diff utility)', () => {
 		cy.visit('/wp-admin/admin.php?page=sucuriscan#auditlogs');
 
+		cy.wait(3000);
```

Comment:
Out of curiousity, why the waits?

Reply:
WordPress and PHP are getting faster 🚀   and I want to test environments such as:
* WordPress 4.9 with PHP 7.2
* WordPress 5.5 with PHP 7.2 
* Etc...
Older versions of WordPress were slower, and that's why I added those. For this PR I removed testing with different versions of WordPress because I hit an error with GitHub Actions, I'll add those in another PR.

Reply:
@imgerson [According to the official documentation](https://docs.cypress.io/guides/references/best-practices#Unnecessary-wait-for-cy-visit), `wait` for `cy.visit()` is not necessary.

Reply:
@gvre @bfeddern-godaddy thanks so much for the feedback 💯. I went over the tests again and the wait exp were not necessary so I removed them.

Reply:
Sorry for coming back to this one @gvre, I ran the tests a few more times and discovered that they were failing inconsistently. For the test `can ignore and unignore false positives (integrity diff utility)` we need to wait for the integrity check to finish, so I addressed that with https://github.com/Sucuri/sucuri-wordpress-plugin/pull/117/commits/4a150e4bfe24fbf9ed8bf37e7c0470991857a137. We now wait for the request to finish, instead of the more general `cy.wait()`.